djimaging/tables/response/orientation.py

```diff
@@ -93,11 +95,12 @@ def get_time_dir_kernels(sorted_responses, dt):
     dir_component *= s
 
     # determine which entries correspond to the first second, assuming 4 seconds presentation time
-    first_second_idx = np.maximum(int(np.round(1. / dt)), 1)
+    first_second_idx = np.maximum(int(np.floor(1.0 / dt)), 1)
     time_component -= np.mean(time_component[:first_second_idx])
-    time_component = time_component / np.max(abs(time_component))
+    time_component = time_component / np.max(np.abs(time_component))
 
-    dir_component = math_utils.normalize_zero_one(dir_component)
+    dir_component = dir_component - np.min(dir_component)
+    dir_component = dir_component / np.max(dir_component)
```

Comment:
Original code did not normalise the dir_component to [0, 1], but just did this minmax